tests/processes/ProcessingTest.py

```diff
@@ -131,6 +131,13 @@ def test_run(self):
                          ') at 0x[0-9a-fA-F]+>'.format(hex(global_result.id)))
 
     def test_empty_run(self):
+        execute_section(self.sections['default'],
+                        [],
+                        [],
+                        lambda *args: self.result_queue.put(args[2]),
+                        None,
+                        self.log_printer,
+                        console_printer=self.console_printer)
```

Comment:
You need to verify the exception has been raised using `with assertRaisesRegex()` Se the unitest docs for more info (https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertRaisesRegex)